package.json

```
@@ -96,7 +97,7 @@
     "lint": "npm run _lint:check",
     "prepack": "npm run build",
     "prepare-pr": "npm test && npm run _api:update && npm run _docs:generate && npm run _format:fix && npm run _lint:fix",
```

```suggestion
    "prepare-pr": "npm run build && npm test && npm run _api:update && npm run _docs:generate && npm run _format:fix && npm run _lint:fix",
```

Since `npm run build` has been removed from the `test` script it needs to be added to the `prepare-pr` script. The doc tasks won't work right if there is no `dist` directory.